JapaneseFilter/sections/css_extended.txt

```diff
@@ -82,6 +82,7 @@ mindhack2ch.com#?#.article-body > .ad:has(> .adsbygoogle)
 mindhack2ch.com#?#.article-body > .rss:has(> script)
 mindhack2ch.com#?#.article-post-bottom > .rss-article:has(> .rss-article-item)
 !
+asahi.com#?##RelatedNews > .InnerArticleList > li:has(> div[id^="div-gpt-ad"])
```

Comment:
There is missed tag `ul` in `.InnerArticleList`. We avoid tags if it's necessary.

Reply:
You mean I should add `ul`?

Reply:
Yes.

Reply:
Added.